easybuild/easyconfigs/a/ARPACK/ARPACK96-goolf-1.5.14.eb

```diff
@@ -0,0 +1,37 @@
+# This file is an EasyBuild config for ARPACK
+# Author: Bart Verleye
+
+easyblock='MakeCp'
+
+name = 'ARPACK'
+version = '96'
+
+homepage = 'http://www.caam.rice.edu/software/ARPACK'
+description = """ ARPACK is a collection of Fortran77 subroutines designed to 
+                solve large scale eigenvalue problems.  """
+
+toolchain = {'name': 'goolf', 'version': '1.5.14'}
+toolchainopts = {'openmp': False}
+
+sources = ['%(namelower)s%(version)s.tar.gz',
+            'patch.tar.gz',
+
+]
+source_urls = ['http://www.caam.rice.edu/software/ARPACK/SRC/']
+
+buildopts = ['lib']
+
+patches = ['ARmake-96.inc.patch']
+
+files_to_copy = [
+    (["/tmp/libarpack_linux.a"], 'lib'),
```

Comment:
well, hardcoding `/tmp` isn't going to work... what if two people try to build ARPACK on the same system?